tests/yam/migration/setup_registration.pm

```diff
@@ -19,8 +20,12 @@ sub run {
     @filter{@addons_drop} = {};
 
     select_console('root-console');
+    script_run('SUSEConnect -d', proceed_on_failure => 1, timeout => 300);
+    foreach my $addon (@addons_drop) {
```

Comment:
Why not re-use the existing function of deregister_dropped_modules while add new code here?

Reply:
This function use `DROPPED_MODULES`, but our test suite use `SCC_ADDONS_DROP	` and we want to remove the release package with addon and we need to check if it's existed before remove.